### PR TITLE
Release v1.10.1

### PR DESCRIPTION
**Pull request by rashmigottipati**
Release PR for v1.10.1

### CODE REVIEW

**Review by estroz — APPROVED** [review]
/lgtm

**Comment by varshaprasad96** [comment]
/hold 
sorry, this https://github.com/operator-framework/operator-sdk/pull/5130 needs to go in

File: changelog/generated/v1.10.1.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## v1.10.1
+
+### Additions
+
+- (go/v3, ansible/v1, helm/v1) Added `containerPort` protocol field required for server-side apply in manifests. ([#4863](https://github.com/operator-framework/operator-sdk/pull/4863))
+
+### Changes
+
+- (go/v3, ansible/v1, helm/v1) Increased resource limits for controller manager. ([#4863](https://github.com/operator-framework/operator-sdk/pull/4863))
```
Comment (asmacdo) [inline changelog/generated/v1.10.1.md:9]: Optional Nit 
```suggestion
- (go/v3, ansible/v1, helm/v1) Increased default resource limits for controller manager. ([#4863](https://github.com/operator-framework/operator-sdk/pull/4863))
```

**Review by asmacdo — APPROVED** [review]
/lgtm

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

**Review by theishshah — APPROVED** [review]

File: website/content/en/docs/upgrading-sdk-version/v1.10.1.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    test: manifests generate fmt vet envtest ## Run tests.
+    - go test ./... -coverprofile cover.out
+    + KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+  ```
+
+_See [#4863](https://github.com/operator-framework/operator-sdk/pull/4863) for more details._
+
+## (go/v3, ansible/v1, helm/v1) Add `containerPort` protocol field in manifests.
+
+In `go/v3`, `ansible/v1`, `helm/v1` projects, specify the `containerPort` protocol in the following files: - `config/default/manager_auth_proxy_patch.yaml` - `config/rbac/auth_proxy_service.yaml`
```
Comment (varshaprasad96) [inline website/content/en/docs/upgrading-sdk-version/v1.10.1.md:23]: Sry, last min formatting nits!
```suggestion
In `go/v3`, `ansible/v1`, `helm/v1` projects, specify the `containerPort` protocol in the following files: 
- `config/default/manager_auth_proxy_patch.yaml`
- `config/rbac/auth_proxy_service.yaml`
```

**Pull request merged**